packages/desktop/components/AccountManagementList.svelte

```diff
@@ -27,42 +32,61 @@
 
 <left-pane class="flex flex-col w-1/3">
     <Pane height={Height.Full}>
-        <left-pane-container class="flex flex-col space-y-10 h-full">
-            <Text type={TextType.h2}>{localize('views.accountManagement.list.title')}</Text>
-            <list-wrapper class="flex flex-col space-y-2">
-                {#each allOutputs as output, index}
-                    <ClickableTile onClick={() => onAccountClick(output)} selected={isSelected(output)}>
-                        <div class="flex flex-col space-y-4">
-                            <div class="flex space-x-2">
-                                <Text
-                                    type={TextType.h5}
-                                    color="gray-800"
-                                    darkColor="gray-500"
-                                    fontWeight={FontWeight.semibold}
+        <left-pane-container class="flex flex-col space-y-4 h-full scrollable-y">
```

Comment:
this should be fixed, it's cut off :police_car: 
![image](https://github.com/iotaledger/firefly/assets/72078237/c2a03533-c115-4f8f-b3a5-71b685b501f7)

suggestion--> `<list-wrapper class="flex flex-col space-y-2 px-2">`